spec/features/admin/sale_prices_spec.rb

```diff
@@ -78,14 +78,12 @@
       find('.flatpickr-minute').set(32)
 
       fill_in('sale_price_value', with: 32.33)
-
       click_button('Create')
-
       within all('[data-hook="sale_prices"] [data-hook="prices_row"]').last do
         expect(page).to have_content(variant.descriptive_name)
         expect(page).to have_content(32.33)
         expect(page).to have_content(country.name)
-        expect(page).to have_content(variant.prices.last.currency)
+        expect(page).to have_content(variant.prices.first.currency)
```

Comment:
Unless I'm missing something, this order isn't guaranteed. Either way, the tests should know the currency, so why don't we just make the assertion against the literal?

```suggestion
        expect(page).to have_content("EUR")
```